smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/DefaultHttpProtocolCustomizations.kt

```diff
@@ -25,4 +26,8 @@ abstract class DefaultHttpProtocolCustomizations : HttpProtocolCustomizable {
     ) {
         // Default implementation is no-op
     }
+
+    override fun renderInternals(ctx: ProtocolGenerator.GenerationContext) {
+        AuthSchemeResolverGenerator().render(ctx)
+    }
```

Comment:
This is the entry point for code-generating auth scheme resolver & auth scheme resolver parameters of a service.